Factorial/Factorial.java

```diff
@@ -0,0 +1,22 @@
+/* This code is used to find factorial 
+of large numbers*/
+import java.util.*;
+import java.lang.*;
+import java.math.*;
+class Factorial
+{
+	public static void main (String[] args) throws java.lang.Exception
+	{
+		    Scanner s = new Scanner(System.in);
+		    int n = s.nextInt();
+		    BigInteger f = new BigInteger("1");
+		    for(int j = 1;j<=n;j++)
+		    {
+		        f = f.multiply(BigInteger.valueOf(j));
+		    }
+		    System.out.println(f);
+	}
```

Comment:
- correct the indentation. keep four spaces instead to tab
- please keep space before and after operators